packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/template.success.tsx

```diff
@@ -500,11 +513,32 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
             </ErrorText>
           </ErrorTextContainer>
         )}
-        <ActionButton {...props} />
+        <ActionButton
+          {...props}
+          isSufficientEthForErc20={isSufficientEthForErc20 || false}
```

Comment:
i dont think you need the `|| false`. It should already be a boolean or at least undefined which is falsey value

Reply:
If i remove the `||false`, i get an error since I set the TS type to boolean so it doesn't like it. What I could do is set the type to boolean or undefined, is that better?

Reply:
what about converting `isSufficientEthForErc20` to boolean like Boolean(isSufficientEthForErc20)?